patterns/complete_constructor.md

```diff
@@ -93,33 +97,79 @@ Thing.prototype.doYourThing = function() {
   ...
 }
 ```
-
-So, without having to read the implementation details of any of the methods, one who would like to use the Thing class would write…
-
+So, without having to read the implementation details of any of the methods, you should be able to use the Thing class like so:
```

Comment:
Don't like this change.  I think `one who would like to` points out that you are writing a class that someone else would like to use.  You are not writing it for yourself.